### PR TITLE
[syncd] Implement bulk set support

### PR DESCRIPTION
**Pull request by stepanblyschak**
Added support for bulk set SAI API for port, PG and queue objects. This allows orchagent to configure those objects in bulk improving configuration time.

### CODE REVIEW

**Comment by DanielaMurin** [comment]
Regarding the tests added in bulk_object.rec, I didn't explicitly see SAI_OBJECT_TYPE_TUNNEL, SAI_OBJECT_TYPE_NEXT_HOP, or SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER.
Are these object types included in the testing?

Thanks

File: syncd/Syncd.cpp [hunk 1]
```diff
@@ -2009,6 +2009,61 @@ sai_status_t Syncd::processBulkOidCreate(
...
+                                objectRids.data(),
+                                attr_list.data(),
+                                mode,
+                                statuses.data());
+
+    if (status == SAI_STATUS_NOT_IMPLEMENTED || status == SAI_STATUS_NOT_SUPPORTED)
+    {
+        SWSS_LOG_ERROR("bulkSet api is not implemented or not supported, object_type = %s",
+                sai_serialize_object_type(objectType).c_str());
+        return status;
```
Comment (kcudnik) [inline syncd/Syncd.cpp:2061]: return not needed here

**Comment by kcudnik** [comment]
please satisfy code coverage

File: syncd/Syncd.cpp (hunk 1 above)
Comment (stepanblyschak) [reply syncd/Syncd.cpp]: Fixed

**Comment by stepanblyschak** [comment]
> please satisfy code coverage

@kcudnik I added tests to tests/BCM56850/bulk_object.rec. I checked that the added code is covered. Does coverage report account for tests/BCM56850/bulk_object.rec.

**Comment by stepanblyschak** [comment]
> Regarding the tests added in bulk_object.rec, I didn't explicitly see SAI_OBJECT_TYPE_TUNNEL, SAI_OBJECT_TYPE_NEXT_HOP, or SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER. Are these object types included in the testing?
> 
> Thanks

Removed those, instead added PG, QUEUE objects and added test for them

**Comment by kcudnik** [comment]
> > Regarding the tests added in bulk_object.rec, I didn't explicitly see SAI_OBJECT_TYPE_TUNNEL, SAI_OBJECT_TYPE_NEXT_HOP, or SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER. Are these object types included in the testing?
> > Thanks
> 
> Removed those, instead added PG, QUEUE objects and added test for them

It should, if not then you would need to add explicit unit tests

**Review by kcudnik — DISMISSED** [review]

**Comment by kcudnik** [comment]
pleae fix build errors

**Comment by stepanblyschak** [comment]
Fails on:

```
Dockerfile:12
--------------------
  11 |     RUN dpkg --purge libswsscommon python3-swsscommon sonic-db-cli libsaimetadata libsairedis libsaivs syncd-vs swss sonic-eventd libdashapi
  12 | >>> RUN dpkg -i /debs/libdashapi_1.0.0_amd64.deb \
  13 | >>>             /debs/libswsscommon_1.0.0_amd64.deb \
  14 | >>>             /debs/python3-swsscommon_1.0.0_amd64.deb \
  15 | >>>             /debs/sonic-db-cli_1.0.0_amd64.deb \
  16 | >>>             /debs/libsaimetadata_1.0.0_amd64.deb \
  17 | >>>             /debs/libsairedis_1.0.0_amd64.deb \
  18 | >>>             /debs/libsaivs_1.0.0_amd64.deb \
  19 | >>>             /debs/syncd-vs_1.0.0_amd64.deb \
  20 | >>>             /debs/swss_1.0.0_amd64.deb
  21 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c dpkg -i /debs/libdashapi_1.0.0_amd64.deb             /debs/libswsscommon_1.0.0_amd64.deb             /debs/python3-swsscommon_1.0.0_amd64.deb             /debs/sonic-db-cli_1.0.0_amd64.deb             /debs/libsaimetadata_1.0.0_amd64.deb             /debs/libsairedis_1.0.0_amd64.deb             /debs/libsaivs_1.0.0_amd64.deb             /debs/syncd-vs_1.0.0_amd64.deb             /debs/swss_1.0.0_amd64.deb" did not complete successfully: exit code: 1

##[error]Bash exited with code '1'.
```

I see same failure on other PRs.

**Comment by kcudnik** [comment]
1.635 Configuration file '/etc/swss/config.d/netbouncer.json', does not exist on system.
1.

this must be some recent change to swss or swss common

**Comment by stepanblyschak** [comment]
@kcudnik Can we merge?

**Review by kcudnik — APPROVED** [review]

**Comment by kcudnik** [comment]
pleae provide description why this change is made

**Comment by stepanblyschak** [comment]
@kcudnik done

**Pull request merged**